CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ the default sensuctl configuration.
 - Default `event.entity.entity_class` to `proxy` in the POST/PUT `/events` API.
 - Proxy entities are now automatically created when events are published with an
 entity that does not exist.
+- Fixed event payload validation on the backend events API to validate the payload with the URL parameters on the /events/:entity/:check endpoint and reject events that do not match.
```

Comment:
Thanks Hillary!

Can you please move this under the `## Fixed` header? Additionally, you will need to sign your commits with DCO in order to avoid the DCO bot triggering a CI failure. You can find instructions on DCO in the project README.md.